tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot

```diff
@@ -284,6 +356,29 @@ Verify "RHODS Probe Success Burn Rate" Alerts Are Not Firing And Continue On Fai
 Skip Test If Alert Is Already Firing
     [Documentation]    Skips tests if ${alert} is already firing
     [Arguments]    ${pm_url}    ${pm_token}    ${rule_group}    ${alert}    ${alert-duration}=${EMPTY}
-    ${alert_is_firing}=    Run Keyword And Return Status    Alert Should Be Firing
+    ${alert_is_firing} =    Run Keyword And Return Status    Alert Should Be Firing
     ...    ${pm_url}    ${pm_token}    ${rule_group}    ${alert}    ${alert-duration}
     Skip If    ${alert_is_firing}    msg=Test skiped because alert "${alert} ${alert-duration}" is already firing
+
+Provoke Tensorflow Image Build Failure
+    [Documentation]    Starts New Build after some time it kill the build and return name of failed build
+    ${build} =    Search Last Build    namespace=redhat-ods-applications    build_name_includes=tensorflow
+    Delete Build    namespace=redhat-ods-applications    build_name=${build}
+    ${failed_build_name} =    Start New Build    namespace=redhat-ods-applications
+    ...    buildconfig=s2i-tensorflow-gpu-cuda-11.4.2-notebook
+    ${pod_name} =   Search Pod  namespace=redhat-ods-applications  pod_start_with=${failed_build_name}
+    Wait Until Build Status Is    namespace=redhat-ods-applications    build_name=${failed_build_name}
+    ...    expected_status=Running
+    Run Command In Container    namespace=redhat-ods-applications    pod_name=${pod_name}[0]
```

Comment:
This could be a keyword

```
Kill Container In Pod
[Documentation]  Kills abruptly a running container in a pod executing `bin/kill 1` inside it 
[Arguments]  ${namespace}  ${pod}  ${container}
```

Reply:
Done